### PR TITLE
Made the media view changes forced update

### PR DESCRIPTION
**Pull request by yasmeensalah**

### CODE REVIEW

File: modules/varbase_features/varbase_media/varbase_media.install [hunk 1]
```diff
@@ -79,3 +79,14 @@ function varbase_media_update_8006() {
...
+/**
+ * Issue #2905259: Added ALT and TITLE columns for images.
+ *                 In Media content page admin/content/media.
+ */
+function varbase_media_update_8007() {
+  $config = \Drupal::configFactory()->getEditable('views.view.media');
+  $config_file = \Drupal::root() . '/' . drupal_get_path('module', 'varbase_media') . '/config/install/views.view.media.yml';
+  $data = (array) Yaml::parse($config_file);
+  $config->setData($data)->save(TRUE);
+}
```
Comment (Natshah) [inline modules/varbase_features/varbase_media/varbase_media.install:92]: Please add one empty line at the end of all files "Drupal standard coding"

Comment (yasmeensalah) [reply modules/varbase_features/varbase_media/varbase_media.install:92]: There is an empty line at the end of this file!

Comment (Natshah) [reply modules/varbase_features/varbase_media/varbase_media.install:92]: Ok then. Good Job!

**Review by Natshah — APPROVED** [review]
Approved. Good Job!

**Pull request merged**